src/components/User.tsx

```diff
@@ -10,22 +11,34 @@ const User = () => {
 
   switch (oidcUserLoadingState) {
     case OidcUserStatus.Loading:
-      return <p>User information loading...</p>;
+      return (
+        <div className="App-navbar__item App-navbar__item--loading">
+          <UserIcon />
+          Loading…
+        </div>
+      );
     case OidcUserStatus.Unauthenticated:
       return (
         <button
           type="button"
-          className="btn btn-primary"
+          className="App-navbar__item"
```

Comment:
Do we want to use the new `<Button>` component here?

Reply:
I did think about that, but I don't _think_ I want to use the button styling up here. Maybe… maybe so though. Let me play with it! (It would certainly let me remove some stuff, as I had to tweak the navbar-item to make it button-compatible.)